src/mokka/src/integration-test/java/pl/hycom/mokka/stubbing/transformer/GroovyResponseTransformerIT.java

```diff
@@ -0,0 +1,83 @@
+package pl.hycom.mokka.stubbing.transformer;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static io.restassured.RestAssured.expect;
+
+/**
+ * @author Bartosz Kuron (bartosz.kuron@hycom.pl)
+ */
+@DirtiesContext
+@ActiveProfiles("test")
+@AutoConfigureTestDatabase
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class GroovyResponseTransformerIT {
+
+    private static final String API_URL = "test-body";
+    private static final String CONTENT_TYPE_KEY = "Content-Type";
+    private static final String CONTENT_TYPE_VALUE = "text/plain";
+
+    @Value("${wiremock.test.httpPort}")
+    private int wiremockHttpPort;
+
+    WireMockServer wm;
```

Comment:
WireMockServer should be injected (will be provided by WireMockServerConfiguration)